client/components/checklist/task.js

```diff
@@ -95,19 +79,31 @@ export class ChecklistTask extends PureComponent {
 						</small>
 					) }
 				</div>
-				<Focusable
+				<ToggleContainer
 					className="checklist__task-icon"
-					onClick={ this.handleToggle }
-					aria-pressed={ completed ? 'true' : 'false' }
+					onClick={ onDismiss }
+					aria-pressed={
+						/* eslint-disable-next-line no-nested-ternary */
+						onDismiss ? ( completed ? 'true' : 'false' ) : undefined
```

Comment:
Calling out for reviewers that ternaries are nested here in violation of eslint rules.

_My opinion_ is that this is quite a clear way to declare these props, but please raise any objections and I'll work out a change to drop the ternary nesting.

Reply:
One thing you could consider is inlining the entire `<ToggleContainer />`, which might make things actually more readable:

```jsx
{ onDismiss
	? <Focusable
		onClick={ onDismiss }
		aria-pressed={ !! completed } // Do we really need the bool strings ('true'/'false') here?
	/>
	: <div />
}
```